tests/ecmp/test_fgnhg.py

```diff
@@ -21,12 +25,13 @@
 PREFIX_IPV6_LIST = [u'fc:05::/128', u'fc:06::/128', u'fc:07::/128']
 FG_ECMP_CFG = '/tmp/fg_ecmp.json'
 USE_INNER_HASHING = False
+WARM_BOOT_TESTING = False
 NUM_FLOWS = 1000
 ptf_to_dut_port_map = {}
 
 pytestmark = [
     pytest.mark.topology('t0'),
-    pytest.mark.asic('mellanox'),
+    #pytest.mark.asic('mellanox'),
```

Comment:
Please explain this change.

Reply:
Inadvertent change as we were cross-platform testing.. have uncommented